backend-schedule/src/main/java/eu/socialedge/hermes/backend/schedule/ResourceMappingsSupport.java

```diff
@@ -0,0 +1,69 @@
+/*
+* Hermes - The Municipal Transport Timetable System
+* Copyright (c) 2017 SocialEdge
+* <p>
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* <p>
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*/
+package eu.socialedge.hermes.backend.schedule;
+
+import lombok.val;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.data.rest.core.mapping.ResourceMappings;
+import org.springframework.data.rest.webmvc.BaseUri;
+import org.springframework.data.rest.webmvc.spi.BackendIdConverter;
+import org.springframework.plugin.core.PluginRegistry;
+import org.springframework.stereotype.Component;
+import org.springframework.web.util.UriTemplate;
+
+import java.io.Serializable;
+import java.net.URL;
+
+@Component
+public class ResourceMappingsSupport {
```

Comment:
Please add javadoc with description what this magic is for.
Btw maybe "ResourceElements" will fit more, coz ID is an element of a resource 👍  Support and Mapping words are somewhat confusing imho